frontend/src/core/loaders/components/WaveLoader.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+
+import React from 'react';
+
+import './WaveLoader.css';
+
+export const WaveLoader = (): React.Node => (
+    <div id="project-load" className="overlay">
```

Comment:
In order to make modules as abstract as possible, I would do 2 things here: first remove the `id` and make it a `class` instead, so that it can be reused more easily if we need to, and rename it to not be `project-load`, because that is too specific to the use case, and not to the nature of this component. I'd go with a simple `wave-loader` class name.